cmd/incusd/instance_exec.go

```diff
@@ -478,6 +478,13 @@ func (s *execWs) Do(op *operations.Operation) error {
 					}()
 				}
 
+				if i == execWSStderr {
+					// Consume websocket pings
```

Comment:
I'd probably expand a bit on the comment so we don't risk removing this :)

```
// Consume data from stderr too to avoid a situation where we hit an inactivity timeout on stderr during long exec sessions.
```